superglue/package.json

```
@@ -8,8 +8,10 @@
     "clean": "rm -rf ./dist",
     "copy:package": "cat ./package.json | jq 'del(.scripts)' > dist/package.json",
     "copy:readme": "cp ../README.md dist/",
-    "build:index": "babel lib -d dist",
-    "build": "npm run clean && npm run build:index && npm run copy:package && npm run copy:readme",
+    "build": "npm run clean && npm run build:js && npm run copy:package && npm run copy:readme",
+    "build:js": "babel lib --out-dir dist --extensions \".ts,.tsx\" --source-maps inline",
```

Good eye with the source maps. To keep parity with main, lets remove the source maps option for now.

```suggestion
    "build:js": "babel lib --out-dir dist --extensions \".ts,.tsx\"",
```

---

Done!